README.md

```diff
@@ -24,16 +32,11 @@ $ docker run --restart=always \
 	--name=shifthelper_mysql \
 	mysql
 ```
```

Comment:
this is supposed to be a test mysql DB right? So is it necessary to do all these security measures? 

Oh, Is the mysql inside this docker container by default running on port 3306 and thus open to the outside world? Then I get this